patches/@blockprotocol+graph+0.3.1-canary-20231207113055.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/node_modules/@blockprotocol/graph/dist/cjs/codegen.cjs b/node_modules/@blockprotocol/graph/dist/cjs/codegen.cjs
+index 8618e27..050e7ca 100644
+--- a/node_modules/@blockprotocol/graph/dist/cjs/codegen.cjs
++++ b/node_modules/@blockprotocol/graph/dist/cjs/codegen.cjs
+@@ -4,7 +4,7 @@ var path = require('node:path');
+ var slim = require('@blockprotocol/type-system/slim');
+ var jsonSchemaToTypescript = require('json-schema-to-typescript');
+ var fs = require('node:fs/promises');
+-var fetch = require('node-fetch');
++// PATCH: rely on native fetch
```

Comment:
Just so I'm aware, which change in this PR required this? Maybe worth mentioning in the comment

Reply:
Done in https://github.com/hashintel/hash/pull/3714/commits/3f5cae97576f20eb68b67a2fc3f950f742d13e32